src/main/java/com/sleekbyte/tailor/common/Severity.java

```diff
@@ -23,8 +23,7 @@ public String toString() {
     /**
      * Exception thrown when invalid severity is provided.
      */
-    public static class IllegalSeverityException extends Exception {
-    }
+    public static class IllegalSeverityException extends Exception {}
```

Comment:
Can you move this exception declaration after the member variables near the top of the file.